runtimes/node/versions/latest/src/server.js

```diff
@@ -2,233 +2,289 @@ const micro = require("micro");
 const { buffer, send } = require("micro");
 const Logger = require("./logger");
 
-const USER_CODE_PATH = '/usr/local/server/src/function';
+const USER_CODE_PATH = "/usr/local/server/src/function";
 
 const server = micro(async (req, res) => {
-    const logger = new Logger(req.headers[`x-open-runtimes-logging`], req.headers[`x-open-runtimes-log-id`]);
+	const logger = new Logger(
+		req.headers[`x-open-runtimes-logging`],
+		req.headers[`x-open-runtimes-log-id`],
+	);
 
-    try {
-        await action(logger, req, res);
-    } catch (e) {
-        logger.write(e, Logger.TYPE_ERROR);
+	try {
+		await action(logger, req, res);
+	} catch (e) {
+		logger.write(e, Logger.TYPE_ERROR);
```

Comment:
The default indentation for js seems to be a bit too much . Can we reduce it?

Reply:
It's same for Bun. It's 1 tab. I think it's okay, that's same as 4 spaces. And There isn't clear winner between 2 and 4 spaces.

Reply:
I guess GitHub renders tab badly. Switched to spaces in biome.js config file